server/pulp/server/db/model/__init__.py

```diff
@@ -563,13 +566,15 @@ def pre_save_signal(cls, sender, document, **kwargs):
         with FileStorage() as storage:
             storage.put(document, document._source_location)
 
-    def set_content(self, source_location):
+    def set_content(self, source_location, downloaded=True):
```

Comment:
This method is used to copy downloaded content into the proper location in the platform.  How can this ever be FALSE?